flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/FlinkConfParser.java

```diff
@@ -139,7 +139,6 @@ public StringConfParser defaultValue(String value) {
     }
 
     public String parse() {
-      Preconditions.checkArgument(defaultValue != null, "Default value cannot be null");
```

Comment:
why do we remove this line? My understanding of code is that `parseOptional` should be used if default can be null

Reply:
The default value for `AVRO_COMPRESSION_LEVEL_DEFAULT`, `PARQUET_COMPRESSION_LEVEL_DEFAULT` is `null`, so we do not have default values there

Reply:
I missed this... Thanks for catching!